### ISSUE TITLE
No module after installing with pip.

### ISSUE THREAD

**Issue by vindolin** [xxxtonixxx/pyPasswordGenerator#1]
Pip seems to install the package but after that python complains:

"ImportError: No module named pyPassGenerator"

**Comment by tonivj5** [comment]
Hi @vindolin ! Thank's for your report :smile: 

I think I have fixed (v1.0.2). Can you test it? :+1:

**Comment by vindolin** [comment]
Hi, sorry I'm late :)
Nope, still doesn't want to install:

```
Collecting pypassgenerator
  Downloading pyPassGenerator-1.0.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-QOh9qo/pypassgenerator/setup.py", line 33, in <module>
        "Topic :: Utilities",
      File "/usr/lib/python2.7/distutils/core.py", line 124, in setup
        dist.parse_config_files()
      File "/usr/lib/python2.7/distutils/dist.py", line 390, in parse_config_files
        parser.read(filename)
      File "/usr/lib/python2.7/ConfigParser.py", line 305, in read
        self._read(fp, filename)
      File "/usr/lib/python2.7/ConfigParser.py", line 546, in _read
        raise e
    ConfigParser.ParsingError: File contains parsing errors: setup.cfg
        [line  2]: '    description-file = README.md\n'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-QOh9qo/pypassgenerator/
```

**Comment by tonivj5** [comment]
Thank you for your report. I think it is already fixed in v1.0.3 version.

You can reinstall from pip. I hope your feedback :-)

NOTE: I downloaded the package from other computer and I did not have problem... But, I have 8.1.2-1 version of pip in both computers...

**Comment by vindolin** [comment]
Thanks, works! :)
